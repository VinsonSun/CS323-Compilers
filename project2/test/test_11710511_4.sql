int test4() {
   int a = 1;
   int a = 2;
}
int test4() {

}