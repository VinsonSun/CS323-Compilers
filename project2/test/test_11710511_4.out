Error type 5 at Line 2: unmatching type on both sides of assignment
Error type 5 at Line 3: unmatching type on both sides of assignment
